src/helpers.ts

```diff
@@ -0,0 +1,112 @@
+import { Person } from './types';
+import { FilterParams } from './types/FilterParams';
+import { Sex } from './types/Sextype';
+import { SortOrder } from './types/SortOrder';
+import { SortType } from './types/SortType';
+
+const NOT_SET_VALUE = '-';
+
+export const getPeopleWithParents = (people: Person[]) => {
+  return people.map(person => {
+    const mother = people.find(mom => person.motherName === mom.name);
+    const father = people.find(dad => person.fatherName === dad.name);
```

Comment:
At the time of the search, you don't yet know if it will be `dad` or `mom`
```suggestion
    const mother = people.find(({ name })=> person.motherName === name);
    const father = people.find(({ name }) => person.fatherName === name);
```